### DISCUSSION TITLE
Access to requested resource is denied after using Quick Start Deployment

### DISCUSSION THREAD

**Discussion by abduleviva** [amzn/selling-partner-api-models#3069] [General]
I cannot understand why I get "Access to requested resource is denied" when I call https://sellingpartnerapi-na.amazon.com/orders/v0/orders/?CreatedAfter=2022-09-26&MarketplaceIds=ATVPDKIKX0DER&MaxResultsPerPage=1

I followed all the steps in the Quick Start Documentation and I can successfully simulate the SPAPI policy with the user. I entered the role ARN in the application and I read all the issues/fixes pertaining to this error here. Yet, I still haven't found an answer to my problem. I'm using an iPaaS called Workato with an Amazon Seller Central connector. 

![FireShot Capture 006 - Amazon seller central adapter - Workato - app workato com](https://user-images.githubusercontent.com/114435085/192360024-0369b5c5-8a11-4188-a736-036dbd98e9a7.png)

**Comment by sungolivia** [comment]
hmm it could be a lot of different things, few things I would check for:

1. Roles in your developer profile and app config, if you added new roles for this then make sure to apply to the app and re-list them ([blog](https://developer-docs.amazon.com/sp-api-blog/docs/choosing-roles-for-developer-profile-and-sp-api-app))
2. issue with your Workato connector? I can't say about this since I haven't used them before
3. General access denied troubleshooting [blog](https://developer-docs.amazon.com/sp-api-blog/docs/troubleshooting-selling-partner-api-authorization-errors) I would read through
